### PR TITLE
Feature/#300 event logging framework placeholder

### PR DESCRIPTION
**Pull request by elnyry**

### CODE REVIEW

File: CentralServices/seq_diagrams/seq-event-9.1.0.plantuml [hunk 1]
```diff
@@ -12,14 +12,27 @@ autonumber
...
 
-box "Central Service" #LightYellow
-    participant CSAPI
-    participant DB
+box "Event Handler Placeholder" #LightGray
+    participant EVENT_HANDLER
 end box
 
-CSAPI -> CSAPI: TODO
+collections "Notification-Topic" as TOPIC_EVENTS
```
Comment (rmothilal) [inline CentralServices/seq_diagrams/seq-event-9.1.0.plantuml]: Should be 'Event-Topic' not 'Notification-Topic'

File: CentralServices/seq_diagrams/seq-prepare-1.1.1.a.plantuml [hunk 2]
```diff
@@ -88,6 +88,7 @@ group Prepare Handler Consume
             TRANS_DAO --> PREP_HANDLER: Return transfer message hash if it exists
             deactivate TRANS_DAO
             alt Transfer exists, hash matches
+                PREP_HANDLER -> TOPIC_EVENTS: Publish event information (for duplicate Prepare) ( Event Handler Consume {9.1.0} )
```
Comment (rmothilal) [inline CentralServices/seq_diagrams/seq-prepare-1.1.1.a.plantuml]: should this not be the same at the Persist Event Information block as above??

group Persist Event Information	     group Persist Event Information
         |||	         |||
         PREP_HANDLER -> TOPIC_EVENTS: Publish event information	         PREP_HANDLER -> TOPIC_EVENTS: Publish event information
         ref over PREP_HANDLER, TOPIC_EVENTS :  Event Handler Consume {9.1.0} \n	         ref over PREP_HANDLER, TOPIC_EVENTS :  Event Handler Consume {9.1.0} \n
         |||	         |||
     end	     end

File: CentralServices/seq_diagrams/seq-event-9.1.0.plantuml (hunk 1 above)
Comment (elnyry) [reply CentralServices/seq_diagrams/seq-event-9.1.0.plantuml]: Thanks, will update

File: CentralServices/seq_diagrams/seq-prepare-1.1.1.a.plantuml (hunk 2 above)
Comment (elnyry) [reply CentralServices/seq_diagrams/seq-prepare-1.1.1.a.plantuml]: updated

File: CentralServices/seq_diagrams/seq-event-9.1.0.plantuml [hunk 3]
```diff
@@ -12,14 +12,59 @@ autonumber
...
+                    }
+                }
+            }
+        }
+    end note
+    note right of EVENT_HANDLER #LightGray
+        The type would be an ENUM with values:
+            [INFO, DEBUG, ERROR, WARN, FATAL, TRACE]
+        Possible values for "action" would be
+            [audit, exception]
```
Comment (mdebarros) [inline CentralServices/seq_diagrams/seq-event-9.1.0.plantuml]: Should this not be caps?

Comment (elnyry) [reply CentralServices/seq_diagrams/seq-event-9.1.0.plantuml]: Updated..

**Pull request merged**